README.md

```diff
@@ -257,3 +257,34 @@ podman save localhost/community -o community.img
 
 Better to upload it to some registry, now I manually loaded it into AAP.
 
+
+### Installing Private Automation Hub
+
+Installation of Private Automation Hub is (currently) automated with four playbooks, including prerequisites.
+
+**1. Prerequisite: Create VM**
+
+```
+ansible-playbook -i localhost -c local -e @../private-lab/secrets.yml -e name=rh-automation-hub-01 ensure-vm-state.yml
```

Comment:
I think i had to change name to short_name in order to get the workflows working. sorry...